src/osd/osd.c

```diff
@@ -71,23 +71,66 @@ static struct view *
 get_next_cycle_view(struct server *server, struct view *start_view,
 		enum lab_cycle_dir dir)
 {
+	bool forwards = dir == LAB_CYCLE_DIR_FORWARD;
+	if (rc.window_switcher.order == WINDOW_SWITCHER_ORDER_OPENED) {
+		/* Build a stable list of candidates in open (creation) order */
+		struct wl_array views; wl_array_init(&views);
```

Comment:
Please use two lines here, we don't usually have two statements within the same line.